CHANGELOG.md

```diff
@@ -36,6 +36,10 @@ To learn more about active deprecations, we recommend checking [GitHub Discussio
 - [v1.1.0](#v110)
 - [v1.0.0](#v100)
 
+## Unreleased
+
+- **JetStream:** JetStream scaler use stream consumer leader metric when clustered ([#2391](https://github.com/kedacore/keda/issues/2391))
```

Comment:
Please move this to `### Improvements` section (alphabetically sorted) and please name the scaler `**Nats Jetstream**`.

Reply:
all set on this, thanks!

Reply:
I see where the e2e test is now...I can probably add a clustered test...should I do that here or in another PR?

Reply:
Please do it here, thanks!